core/domain/question_services_test.py

```diff
@@ -118,3 +118,19 @@ def test_update_question(self):
         self.assertEqual(
             model.question_data_schema_version, question_data_schema_version)
         self.assertEqual(model.language_code, language_code)
+
+    def test_commit_log_entry(self):
+        question = question_domain.Question(
+            'dummy', self._create_valid_question_data('ABC'), 1, 'en')
+
+        QUESTION_ID = question_services.add_question(
```

Comment:
This is 'dummy' itself, right? 
Anyways, see the previous comment.

Reply:
This is a variable. I think it should be in small. right?

Reply:
Yeah, make that small also.